guessFinal.js

```diff
@@ -11,79 +9,10 @@ function ask(questionText) {
 
 start();
 
-let min=0;
-let max= parseInt(process.argv[2]) ||100;
-async function start(){
-
-  console.log(`\nPick a number between 1 and ${process.argv[2] ||100} and I'll try to guess it.\n`)
```

Comment:
You've already stored `process.argv[2] || 100` with the variable `max` so you could just use that here.